Rust/src/rcf.rs

```diff
@@ -262,18 +303,71 @@ where
                         centrality,
                         &new_point,
                         &self.point_store,
-                        ignore_mass,
-                        score_seen,
-                        score_unseen,
-                        damp,
-                        normalizer,
+                        visitor_info
                     )
                 })
                 .collect()
         };
-        list.sort_by(|o1,o2| o1.0.cmp(&o2.0));
+        list.sort_by(|&o1,&o2| o1.2.partial_cmp(&o2.2).unwrap());
         list
     }
+
+    pub fn simple_traversal<NodeView,V,R,S>(
+        &self,
+        point: &[f32],
+        parameters : &[usize],
+        visitor_info: &VisitorInfo,
```

Comment:
Why is VisitorInfo it's own top-level struct that has to be passed into every traversal? Why aren't these parameters in the Visitor struct itself?

Reply:
Saves a lot of space ...